keyboards/dumbpad/dumbpad.c

```diff
@@ -15,15 +15,47 @@
  */
 #include "dumbpad.h"
 
+void keyboard_pre_init_user(void) {
+  // Set the layer LED IO as outputs
+  setPinOutput(LAYER_INDICATOR_LED_0);
+  setPinOutput(LAYER_INDICATOR_LED_1);
+}
+
+void shutdown_user() {
+  // Shutdown the layer LEDs
+  writePinLow(LAYER_INDICATOR_LED_0);
+  writePinLow(LAYER_INDICATOR_LED_1);
+}
+
+uint32_t layer_state_set_user(uint32_t state) {
+  // Layer LEDs act as binary indication of current layer
+  uint8_t layer = biton32(state);
+  writePin(LAYER_INDICATOR_LED_0, layer & 0b1);
+  writePin(LAYER_INDICATOR_LED_1, (layer >> 1) & 0b1);
+  return state;
```

Comment:
In addition with the change above: 
```suggestion
  return layer_state_set_user(state);
```